frontend/src/views/Lobby.tsx

```diff
@@ -63,16 +80,25 @@ function LobbyPage() {
       setStateFromRoom(JSON.parse(result.body));
     };
 
+    const startGameCallback = () => {
+      history.replace('/game');
```

Comment:
I'm actually not sure whether replace or push would be better here, open to hearing thoughts on this. On the one hand, using replace, clicking back will take you directly back to the join/create page which seems like good behavior. On the other hand, I feel like push makes more sense cause you're adding to the history, whereas we've mainly used replace when a user is on a page they're unauthorized for.

Reply:
In that case I think push makes a little more sense if it can be recorded in the history. Do you mean the back button doesn't work with push?

Reply:
I think that was only the case when users are unauthorized to access the page and are immediately redirected elsewhere, cause then if you click back, then it would just automatically redirect you again. In this scenario I don't think that would be a problem (it's also like an edge case so we can always figure it out at a later time).